orchagent/aclorch.cpp

```diff
@@ -4447,7 +4740,7 @@ void AclOrch::doAclTableTask(Consumer &consumer)
             // Add mandatory ACL action if not present
             newTable.addMandatoryActions();
             // validate and create/update ACL Table
-            if (bAllAttributesOk && newTable.validate())
+            if (egrSetDscpStatus && bAllAttributesOk && newTable.validate())
```

Comment:
Why check `egrSetDscpStatus` here? It will break the current code.

Reply:
the egrSetDscpStatus  is by default TRUE. it can only be false if we attempt to create EGR_SET_DSCP table and fail. In such a case creating the MARK_META/V6 table is futile and should fail.
The logic goes like this.
egrSetDscpStatus  = true
if UNDERLAY_SET_DSCP 
     then create EGR_SET_DSCP. failure should set  egrSetDscpStatus  =false
if EGR_SET_DSCP is false then dont create MARK_META/v6
if EGR_SET_DSCP is true then create MARK_META/v6 or other table for which the request came in.